docs/operating-scylla/nodetool-commands/decommission.rst

```diff
@@ -29,5 +29,13 @@ Before you run ``nodetool decommission``:
   request may fail.
   In such a case, ALTER the keyspace to reduce the RF before running ``nodetool decommission``.
 
+It's allowed to invoke ``nodetool decommission`` on multiple nodes in parallel. This will be faster than doing
```

Comment:
We could mention that it's possible to bootstrap nodes during draining. Not sure what is a good place.